source/m2minterfaceimpl.cpp

```diff
@@ -61,7 +61,8 @@ M2MInterfaceImpl::M2MInterfaceImpl(M2MInterfaceObserver& observer,
   _security(NULL),
   _retry_count(0),
   _reconnecting(false),
-  _retry_timer_expired(false)
+  _retry_timer_expired(false),
+  _bootstrapped(true)
```

Comment:
Isn't this a bit confusing initiating to true, does it have to be initiated to true in the beginning when it's not actually bootstrapped?